src/main/java/com/o19s/es/ltr/feature/store/ScriptFeature.java

```diff
@@ -84,7 +90,7 @@ public Query doToQuery(LtrQueryContext context, FeatureSet featureSet, Map<Strin
         Map<String, Object> queryTimeParams = new HashMap<>();
         for (String x : queryParams) {
             if (params.containsKey(x)) {
-                queryTimeParams.put(x, params.get(x));
+                queryTimeParams.put(ltrToScriptParams.get(x), params.get(x));
```

Comment:
this seems to be breaking change as you now force users to declare all their params as part of this new indirection mechanism
This may as well produce a NPE

Reply:
adding the dedicated `extra_script_params` should fix this issue